src/leds.c

```diff
@@ -261,6 +275,33 @@ static void anim_knight_rider(Leds *leds, const LedStrip *strip, const LedBar *b
     }
 }
 
+static void anim_felony(Leds *leds, const LedStrip *strip, float time) {
+    uint32_t color_off = 0x00000000;
+    uint32_t color_red = 0x00ff0000;
+    uint32_t color_blu = 0x000000ff;
```

Comment:
Could you take the two colors from the configuration, so that the effect can be configured in any color combination?

Reply:
Ooh, that's a great idea! Done :+1: